### ISSUE TITLE
Format for targets

### ISSUE THREAD

**Issue by manueldiaz96** [packyan/PyTorch-YOLOv3-kitti#10]
Hi!

I just wanted to ask about what is the final format of the targets when you pass them to the network for training?

From your code I understand that you read the GT from a txt file, but on [datasets.py](https://github.com/packyan/PyTorch-YOLOv3-kitti/blob/master/utils/datasets.py) in line 96 you use `np.readtxt` but the labels begin with a string [(see this example)](https://github.com/packyan/PyTorch-YOLOv3-kitti/blob/master/data/kitti/labels/train/000000.txt). 

Could you please do me the favor of specifying what is the composition of the targets? I know that per image you limit the amount of boxes to 50, so the targets will have shape `[batch_size, 50, 5]`

Are they in the form `[class, center_x_ratio, center_y_ratio, box_width_ratio, box_height_ratio]`?

Where `ratio` means the value in pixels divided by the width of the image (for `center_x` and `box_width`) or by the height of the image (for `center_y` and `box_height`).

Thank you!
